### ISSUE TITLE
您好，请问一下在losloop数据集上按照你的代码跑只有87%的精度，怎么达到您论文中的91%的精度呢，十分感谢！

### ISSUE THREAD

**Issue by searcher623** [lehaifeng/T-GCN#44]

**Comment by martinwhl** [comment]
Hi there, 

Which version of code (TensorFlow/PyTorch) did you use? Did you follow the original hyperparameter settings?

**Comment by martinwhl** [comment]
I notice the problem here. Line 55 of `T-GCN/T-GCN-PyTorch/tasks/supervised.py`

```Python
loss = F.mse_loss(predictions, y)
```

should be 

```Python
loss = self.loss(predictions, y)
```

Codes will get updated soon. The accuracy should reach at least 0.9.

**Comment by martinwhl** [comment]
Codes are updated, closing this issue.
